osquery/main/shell.cpp

```diff
@@ -35,9 +40,17 @@ HIDDEN_FLAG(int32,
 DECLARE_bool(disable_caching);
 }
 
+#ifdef WIN32
+#define IsAtty _isatty
```

Comment:
I'd love some guidance/insight here, but representing small platform differences in APIs with a capitalized (or camel case) symbol seems clowny. Perhaps we can use `platformIsATTY` or something similar? We add the `platform` prefix in other parts of the code base too!